.github/workflows/msal-angular-e2e.yml

```diff
@@ -1,17 +1,19 @@
 # This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+# Will not trigger unless PR is ready for review.
 
 name: msal-angular E2E Tests
 
 on:
   push:
-    branches: 
+    branches:
       - dev
       - master
   pull_request:
-    paths: 
+    types: [opened, reopened, synchronize, ready_for_review]
```

Comment:
`synchronize` triggers on every commit so the rest become redundant here. I think what you're trying to do here is ready_for_review && any of the previous 3 triggers? The ready for review check would need to go below along with the draft check instead to achieve that.

I think I'd also be fine with only the draft check below for now if we can all be diligent about using draft PRs when still in progress.

Reply:
We have to keep at least "synchronize" and "ready_for_review" to kick off e2e workflow once PR is ready for review without pushing new commits to the branch.

Reply:
Ah, my apologies, I misunderstood (was still thinking "approved"), thanks for clarifying.

Reply:
Can we have draft as the default setting when we open a PR for the repo?

Reply:
Github defaults to whatever you personally used last. Don't think there is a repo wide setting.